src/foam/nanos/auth/services.jrl

```diff
@@ -74,3 +74,17 @@ p({
       }
     """
 })
+
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"credentialsDAO",
+  "serve":true,
+  "serviceScript":`
+    return new foam.dao.EasyDAO.Builder(x)
+      .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)
```

Comment:
setFuid(true) for id if not provided.

Reply:
On second thought, let's not.

Reply:
Any reason why not?

Reply:
I was looking at your config and I think the explicit human ids will be the norm: rbc_credentials, but ids have to be unique, so you can't have 
id: rbc_credentials for two different spids. 
So perhaps it should use FUID and also have a 'service' or 'name' property by which we lookup.  

id: fuid
service: String, like 'RBC_EFT'
then find(EQ(Credential.SERVICE, "RBC_EFT")) and a DAO decorator can add where(ENABLED, TRUE) and we already get Spid filtering.